docs/src/12-shadows.stories.mdx

```diff
@@ -0,0 +1,24 @@
+import { Meta, Typeset } from '@storybook/addon-docs/blocks';
+import { TokenTable } from './lib/TokenTable';
+import * as tokens from '@wmde/wikit-tokens';
+import styles from './styles/transition.css';
```

Comment:
```suggestion
```

Alternatively, move the many styles from the component into a file made for that purpose (except for the `boxShadow`, obviously).